netcore/metapackage-llvm.nuspec

```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2010/07/nuspec.xsd">
+    <metadata>
+        <id>Microsoft.NETCore.Runtime.MonoLLVM</id>
+        <version>$VERSION$</version>
+        <description>Internal implementation package not meant for direct consumption.  Please do not reference directly. 
+The Mono runtime, and the base library, called mscorlib. It includes the garbage collector, JIT compiler, base .NET data types and many low-level classes.</description>
+        <authors>Microsoft</authors>
+        <projectUrl>https://www.mono-project.com/</projectUrl>
+        <dependencies>
+            <dependency id="runtime.linux-x64.Microsoft.NETCore.Runtime.MonoLLVM" version="$VERSION$" />
```

Comment:
s/linux-x64/$RID$/ ?

Reply:
@filipnavara only linux will be supported for now :)

Reply:
Is there any reason for that? I'd consume the packages on macOS if they were available. Besides, the other .nuspec has $RID$ already.

Reply:
Oh, wait, the metapackage should reference all the available platforms, right? (embarrased look)

Reply:
exactly, I'll add macos later (windows is not supported at all)